### ISSUE TITLE
Add sketch compose multiplatform image loading library

### ISSUE THREAD

**Issue by panpf** [terrakok/Compose-Multiplatform-Wizard#66]
Sketch is an image loading library designed for Compose Multiplatform and Android View. It is powerful and rich in functions. In addition to basic functions, it also supports GIF, SVG, video thumbnails, Exif Orientation, etc.

https://github.com/panpf/sketch

**Comment by terrakok** [comment]
I agree this is a good library but not for the common wizard. In fact, the Coil is standard at the moment
